code/game/objects/items/devices/laserpointer.dm

```diff
@@ -117,14 +117,18 @@
 
 	//robots
 	else if(iscyborg(target))
-		var/mob/living/silicon/S = target
+		var/mob/living/silicon/robot/S = target
 		log_combat(user, S, "shone in the sensors", src)
 		//chance to actually hit the eyes depends on internal component
 		if(prob(effectchance * diode.rating))
-			S.flash_act(affect_silicon = 1)
-			S.Paralyze(rand(100,200))
-			to_chat(S, "<span class='danger'>Your sensors were overloaded by a laser!</span>")
-			outmsg = "<span class='notice'>You overload [S] by shining [src] at [S.p_their()] sensors.</span>"
+			S.flash_act(affect_silicon = 1, type = /atom/movable/screen/fullscreen/flash/static)
```

Comment:
``type`` is ``/atom/movable/screen/fullscreen/flash/static`` by default, I believe. If that's right, it might be a good idea to not redefine it here, just in-case we change stuff down the line.